x-pack/elastic-agent/pkg/eql/Eql.g4

```diff
@@ -0,0 +1,97 @@
+// eql.g4
```

Comment:
@blakerouse Would be nice to have a link to where this is copied from?

Reply:
@ruflin I didn't copy it, I wrote it. I added on to the Boolexp.g4 that was already present, I assume originally implemented by @ph

Reply:
I did the original implementation before we knew about eql.

I've done a bit more digging, endpoint did create a grammar file for antl4r, in https://github.com/elastic/elasticsearch/blob/master/x-pack/plugin/eql/src/main/antlr/EqlBase.g4 but this its more and less at the same time.

I don't think we should use that directly.

Reply:
Good to have this reference. I good in keeping them separate as there are small variations like the variable syntax but it is expected.